packages/sessions/src/tracker.ts

```diff
@@ -6,6 +6,7 @@ export type PresignedConfig = {
   nextConfig: commons.config.Config
   signature: string
   referenceChainId?: string
+  validateBehavior?: 'ignore' | 'throw'
```

Comment:
Shouldn't this be an argument of `savePresignedConfiguration` instead? Does `PresignedConfigLink` makes any use of it?

Reply:
Seems related to `referenceChainId`. Both are for validation purposes.

Reply:
Not really. referenceChainId specifies the chain to use for signature validation (without it, validation wouldn’t be possible), as it gives meaning to the signature. validateBehavior is not directly tied to the signature itself but acts as a rule for the process involving the signature. This “rule” shouldn’t inadvertently cross into other contexts.